src/components/form/widgets/date-picker.ts

```diff
@@ -53,15 +54,16 @@ export class DatePicker extends React.Component {
         }
     }
 
-    private handleChange(event: CustomEvent<boolean>) {
+    private handleChange(event: CustomEvent<Date>) {
```

Comment:
Oops… I guess I missed this when reviewing the PR for the datepicker widget?

Reply:
Yeah, copy + paste mistake :frowning_face: